common/tests/CMakeLists.txt

```diff
@@ -29,5 +31,14 @@ if(${run_int_tests})
         build_test_folder(gballoc_ll_mimalloc_int)
         build_test_folder(gballoc_ll_win32heap_int) 
         build_test_folder(gballoc_ll_passthrough_int)
+        
+        #this builds and runs the same test for gballoc_hl but with all the implementations of the underlying gballoc_ll.
+        #this scripts sets a variable (GBALOC_LL_IMPL) that is picked as underlying implementatio of gballoc_ll and consequentially as file name (gballoc_ll_${GBALOC_LL_IMPL})
+        foreach(GBALOC_LL_IMPL ${GBALLOC_LL_TYPE_VALUES})
+        #set(GBALOC_LL_IMPL "PASSTHROUGH")

```

Comment:
>#set(GBALOC_LL_IMPL "PASSTHROUGH") [](start = 8, length = 34)

can be removed?

Reply:
Yes, leftovers from CMake debug. Thanks for noticing. :)

---
In reply to: [458465947](https://github.com/Azure/azure-c-pal/pull/12#discussion_r458465947) [](ancestors = 458465947)

Reply:
Removed.

---
In reply to: [458501198](https://github.com/Azure/azure-c-pal/pull/12#discussion_r458501198) [](ancestors = 458501198,458465947)